src/sentry/workflow_engine/handlers/condition/anomaly_detection_handler.py

```diff
@@ -1,16 +1,42 @@
-from typing import Any
-
+from sentry.seer.anomaly_detection.types import (
+    AnomalyDetectionSeasonality,
+    AnomalyDetectionSensitivity,
+    AnomalyDetectionThresholdType,
+)
 from sentry.workflow_engine.models.data_condition import Condition
 from sentry.workflow_engine.registry import condition_handler_registry
-from sentry.workflow_engine.types import DataConditionHandler, WorkflowEventData
+from sentry.workflow_engine.types import (
+    DataConditionHandler,
+    DetectorPriorityLevel,
+    WorkflowEventData,
+)
 
 
 @condition_handler_registry.register(Condition.ANOMALY_DETECTION)
 class AnomalyDetectionHandler(DataConditionHandler[WorkflowEventData]):
     group = DataConditionHandler.Group.DETECTOR_TRIGGER
-    comparison_json_schema = {"type": "boolean"}
+    comparison_json_schema = {
```

Comment:
I needed to add this stuff otherwise I couldn't set up the condition in the test correctly.